docs/source/nlp/tutorials/quickstarts/plot_token_classification.py

```diff
@@ -0,0 +1,175 @@
+   many other checks.
+3. Your model's predictions (see :ref:`nlp__supported_tasks` for info on supported formats). These are needed only for
+   the model related checks, shown in the `Model Evaluation <#model-evaluation>`__ section of this guide.
+
+If you don't have deepchecks installed yet:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install deepchecks[nlp] -U --quiet #--user
+
+Some properties calculated by ``deepchecks.nlp`` require additional packages to be installed. You can
+install them by running:
+
+.. code:: python
+
+    import sys
+    !{sys.executable} -m pip install [nlp-properties] -U --quiet #--user
+
+Setting Up
+==========
+
+Load Data
+---------
+For the purpose of this guide, we'll use a small subset of the
+`SCIERC <http://nlp.cs.washington.edu/sciIE/>`__ dataset:
+
+"""
+from pprint import pprint
+from deepchecks.nlp import TextData
+from deepchecks.nlp.datasets.token_classification import scierc_ner
+
+train, test = scierc_ner.load_data(data_format='Dict')
+pprint(train['text'][0][:10])
+pprint(train['label'][0][:10])
+
+# %%
+#
+# The SCIERC dataset is a dataset of scientific articles with annotations for named entities, relations and coreference.
+# In this example we'll only use the named entities annotations, which are the labels we'll use for our token
+# classification task.
+# We can see that we have the article text itself, and the labels for each token in the text in the
+# :ref:`IOB format <nlp__supported_token_classification>`.
+#
+# Create a TextData Objects
+# -------------------------
+#
+# We can now create a :ref:`TextData <nlp__textdata_object>` object for the train and test dataframes.
+# This object is used to pass your data to the deepchecks checks.
+#
+# To create a TextData object, the only required argument is the tokenized text itself, but passing only the text
+# will prevent multiple checks from running. In this example we'll pass the label and define the task type.
+
+
+train = TextData(tokenized_text=train['text'], label=train['label'], task_type='token_classification')
+test = TextData(tokenized_text=test['text'], label=test['label'], task_type='token_classification')
+
+# %%
+# Calculating Properties
+# ----------------------
+#
+# Some of deepchecks' checks use properties of the text samples for various calculations. Deepcheck has a wide
+# variety of such properties, some simple and some that rely on external models and are more heavy to run. In order
+# for deepchecks' checks to be able to access the properties, they must be stored within the
+# :ref:`TextData <nlp__textdata_object>` object. You can read more about properties in the
+# :ref:`Property Guide <nlp__properties_guide>`.
+
+# properties can be either calculated directly by Deepchecks
+# or imported from other sources in appropriate format
+
+# device = torch.device('cuda' if torch.cuda.is_available() else 'cpu')
```

Comment:
Add a sentence saying that this is how we calculate

Reply:
Don't think it's needed, should be pretty clear for most nlp users and it's long enough already. Added a line for that in the property guide as it was missing.